GetStream/stream-chat-android
# [3863] Fix reply option on gallery screen

### 🎯 Goal

Closes #3863 

Fix reply option in the gallery and moderation actions for the message composer. 

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/9600921/177924270-c8c9bb74-d3b7-4899-bf19-ad0a8becb79b.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/9600921/177924284-e169f403-c28c-4b79-b5b4-fc2ddcbf6e97.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

1. Navigate to the gallery screen
2. Trigger popup menu and select "Reply"
3. Ensure that the composer shows quoted message

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
